examples/nodejs/rate-limiter/get-increment-rate-limiter.ts

```diff
@@ -0,0 +1,48 @@
+import { CacheClient, CacheGet } from "@gomomento/sdk";
+import {
+  AbstractRateLimiter,
+  RATE_LIMITER_CACHE_NAME,
+  RATE_LIMITER_TTL_MILLIS,
+} from "./rate-limiter";
+
+export class GetIncrementRateLimiter extends AbstractRateLimiter {
+  _client: CacheClient;
+  _limit: number;
+
+  constructor(client: CacheClient, limit: number) {
+    super();
+    this._client = client;
+    this._limit = limit;
+  }
+
+  public async acquire(id: string): Promise<boolean> {
```

Comment:
nit: I wonder if a name like `isLimitExceeded` would be a more natural name for a class that provides rate limiting functionality?

Reply:
I actually took this terminology from [guava](https://guava.dev/releases/19.0/api/docs/index.html?com/google/common/util/concurrent/RateLimiter.html), but that's actually a blocking call and waits until a permit is available. So in lieu of that I like your suggestion!